### DISCUSSION TITLE
Missing elements

### DISCUSSION THREAD

**Discussion by ghost** [streetcomplete/StreetComplete#5653] [Q&A] [answered]
Hi ! When I want to add a laundry (i'm in France for example), I can't. Is it normal ?

![image](https://github.com/streetcomplete/StreetComplete/assets/116185937/8572ade4-17cc-45c4-805b-19d3bd303726)
https://github.com/streetcomplete/StreetComplete/blob/46a3605c1337f88d0d62a2e140121d1e6f9ffd03/app/src/main/assets/osmfeatures/default/fr.json#L9953-L9956

**Comment by matkoniecz** [comment, marked as answer]
Shops are available in places overlay (layer icon has orange background), you are currently in things overlay (green background one).

On changing overlay different objects will be shown and can be added.
